src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -308,38 +330,46 @@ Some apps choose to use `Fragment`s as entire Android screens. In these apps, it
 would be reasonable for a `Fragment` to control system chrome like Android's
 status bar, navigation bar, and orientation.
 
-<!-- TODO(mattcarroll): to add back. -->
-<!-- add-flutter-fragment_fullscreen was here-->
+{% asset
+development/add-to-app/android/add-flutter-fragment/add-flutter-fragment_fullscreen.png
```

Comment:
@matthew-carroll it's this PR. I'm going to comment these out for now so I can keep building. Don't forget to put this back along with their assets later.

Reply:
I have a PR up with the assets. Can you approve that PR and leave these tags as-is?

Reply:
SG thanks